contribs/gmf/less/layertree.less

```diff
@@ -30,51 +42,69 @@ gmf-layertree ul .treenode {
     }
   }
 
-  .rightButtons *::after {
-    background: none;
-    color: @map-tools-color;
-    display: inline;
-    font-family: FontAwesome;
+  .rightButtons {
+    float: right;
+
+    *::before,
+    *::after {
+      background: none;
+      position: relative;
+      right: initial;
+    }
+
+    *::after {
+      color: @map-tools-color;
+      display: inline;
+      font-family: FontAwesome;
+    }
+
   }
+
   .state {
     font-family: gmf-icons;
     color: @color;
   }
-  .on .state {
-    color: @special-link;
-  }
+
   .off, .off .legend img {
     opacity: 0.5;
   }
-  .leaf .state::after {
-    content: "\e603";
+
+  .leaf {
+    font-size: smaller;
+    .state::after {
+      content: "\e603";
+    }
   }
 
   .group {
+    font-size: smaller;
     .state::after {
       content: "\e600";
     }
     .layerIcon {
       display: none;
     }
+
+    &.depth-1 {
+      //styling the header of first level groups
+      font-size: small;
```

Comment:
I'd prefer we use bootstrap variables instead of small / smaller. ie. @font-size-base / @font-size-small
Let's use font-size-base for depth-1 groups labels.